src/python/impactx/dashboard/Input/defaults.py

```diff
@@ -38,6 +38,15 @@
     },
 }
 
+CONVERSION_FACTORS = {
+    "meV": 1.0e-9,

```

Comment:
I think we can actually remove milli-eV because we generally deal with energies of keV to TeV (eV is ok to keep).